src/index.ts

```diff
@@ -1,49 +1,55 @@
 #!/usr/bin/env node
-import fs from 'fs-extra';
-import path from 'path';
-import yargs from 'yargs';
-import { compileFromFile } from 'json-schema-to-typescript';
-import getAllFiles from './scripts/get-all-files';
-import generateJsonSchemaValidators from './scripts/generate-json-schema-validators.js';
-
-interface Options {
```

Comment:
interface was an artifact left over from strict typescript refactor that turned out not to be needed, removed.